app/src/main/java/com/github/polypoly/app/base/game/Player.kt

```diff
@@ -0,0 +1,167 @@
+package com.github.polypoly.app.base.game
+
+import com.github.polypoly.app.base.game.bonus_card.InGameBonusCard
+import com.github.polypoly.app.base.game.location.InGameLocation
+import com.github.polypoly.app.base.game.location.LocationBet
+import com.github.polypoly.app.base.user.User
+import kotlin.random.Random
+
+/**
+ * A class that represent a [Player] in a [Game] with his/her info which are specific to the [Game]
+ * @property user The [User] behind the [Player]
+ * @property balance The current balance of the money of the [Player]
+ * @property ownedLocations The list of the [InGameLocation]s owned by the [Player]
+ * @property roundLost Round the [Player] had lost the [Game] if he/she has lost the [Game],
+ * null otherwise
+ */
+data class Player (
+    val user: User = User(),
+    private var balance: Int = 0,
+    private var ownedLocations: List<InGameLocation> = listOf(),
+    private var roundLost: Int? = null,
+) : Comparable<Player> {
+
+    /**
+     * If the player has lost the game (i.e. if the player has no more money)
+     * @return true if the player has lost the game, false otherwise
+     */
+    fun hasLose(): Boolean {
```

Comment:
spelling: `hasLost` (like the doc comment correctly states already)

Reply:
thank you, fix done